README.md

```diff
@@ -28,10 +46,9 @@ To create a `libgmock.a` file in a Linux system follow these instructions in an
 5. Collect both the resulting .o files into an archive file:
     `ar -rv libgmock.a gtest-all.o gmock-all.o`
 
-Congratulations! You now have a `libgmock.a` archive file, include it in the `../build/.lib/` directory and you should be good to go, run `qmake` then `make` in the Tests directory to check if you can compile the tests. Use `ls -a` to show all hidden files (including ones starting with `.`)
+Congratulations! You now have a `libgmock.a` archive file. Create a `Hermes/build/.lib` directory using `mkdir -p Hermes/build/.lib` in the root directory and then place the `libmock.a` archive file in the `Hermes/build/.lib/` directory and you should be good to go. Run `qmake` and then `make` in the `Hermes/src/Tests/` directory to check if you can compile the tests. Use `ls -a` to show all hidden files (including ones starting with `.`).
```

Comment:
I know I didn't do it before, but we should follow the format of new sentences on new lines.